index.html

```diff
@@ -1,16 +1,91 @@
-<!doctype html>
+<!DOCTYPE html>
 <html>
-  <head>
-    <title>Why Blocked?</title>
-    <script src="./example/bundle.js"></script>
-  </head>
-  <body>
-    <h1>Why Was This Site Blocked?</h1>
-    <form>
-      <input id="input"></input>
-      <button class="why" type="submit">Check</button>
-    </form>
-    <br>
-    <p id="result"></p>
-  </body>
+<head>
+	<title>Why Blocked?</title>
+	<script>
+/* Hack: This isn't NodeJS, pretend it is. */
+window.require = function(s){
+	console.debug("Required " + s);
+	if(s === "fastest-levenshtein")
+		return window.module.exports;
+	else if(s === "fast-levenshtein")
+		return window.Levenshtein;
+};
+window.module = {};
+	</script>
+	<!-- Many thanks to the folks at unpkg.com for this wonderful tool. -->
+	<script src="https://unpkg.com/fastest-levenshtein/index.js"></script>
```

Comment:
Can these scripts be inlined/bundled rather than fetched remotely? (besides the security issue - which could be resolved by integrity hashes - this is already broken as this URL currently redirects to a 404 location)